src/capidle/index.html

```diff
@@ -1 +1,35 @@
-this is the homepage for capidle. <a href="../index.html">Back</a>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Capidle</title>
+    <meta name="theme-color" content="#009688" />
+    <meta name="description" content="Like Wordle but for cities 🌏" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+
+    <link rel="stylesheet" href="../main.css" />
+    <link rel="stylesheet" href="./capidle.css" />
+  </head>
+  <body>
+    <nav>
+      <div><a href="../index.html">⬅️</a></div>
+      <div>Capidle</div>
+      <div><a href="#">❔</a></div>
+    </nav>
```

Comment:
I think it would look great if the other 2 mini-games also had a similarly styled navbar

Reply:
I've opened #66 so we can discuss this further